ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/snapshot/entities/Snapshot.java

```diff
@@ -3,12 +3,16 @@
 import java.time.LocalDateTime;
 
 public class Snapshot {
+
 	private final String snapshotId;
 	private final String shape;
 	private final LocalDateTime snapshotUntil;
 	private final String snapshotOf;
+	private final String collectionName;
 
-	public Snapshot(String snapshotId, String shape, LocalDateTime snapshotUntil, String snapshotOf) {
+	public Snapshot(String collectionName, String snapshotId, String shape, LocalDateTime snapshotUntil,
```

Comment:
Stupid thing, but I prefer Id to be the first attribute of an entity. So eternal gratitude if you could change the order here :-)